scripts/paas_app_wrapper.sh

```diff
@@ -22,7 +22,7 @@ case $NOTIFY_APP_NAME in
     ;;
   delivery-worker-sender)
     exec scripts/run_multi_worker_app_paas.sh celery multi start 3 -c 10 -A run_celery.notify_celery --loglevel=INFO \
-    -Q send-sms-tasks,send-email-tasks
+    --logfile=/dev/null --pidfile=/tmp/celery%N.pid -Q send-sms-tasks,send-email-tasks
```

Comment:
It's not just the sender worker that uses up disk space quickly. I assume this double logging is happening on all our celery workers. If that is correct, then we should probably apply this approach to all workers or is there a reason not to do that?

Reply:
True, it's not just the sender worker that uses up space quickly, but it is the only one that has duplicate logs. For the rest of the workers, by default, if you don't define a logfile, celery is printing to stderr [[link]] and we're redirecting the stderr to `/dev/null/`

[link]: https://docs.celeryproject.org/en/3.1/_modules/celery/app/log.html#Logging.setup_task_loggers